awx/main/models/inventory.py

```diff
@@ -424,8 +425,16 @@ def update_computed_fields(self):
             tasks = self.jobs.filter(status="pending")
             for t in tasks:
                 t.task_impact = t._get_task_impact()
-            UnifiedJob.objects.bulk_update(tasks, ['task_impact'])
+            # (method) def bulk_update(
+            #    objs: Iterable[UnifiedJob],
+            #    fields: Sequence[str],
+            #    batch_size: int | None = ...
+            # ) -> int
+            # https://pypi.org/project/django-pg-bulk-update/
+            # Function returns number of updated records.
+            rowcount = UnifiedJob.objects.bulk_update(tasks, ['task_impact'])
         logger.debug("Finished updating inventory computed fields, pk={0}, in {1:.3f} seconds".format(self.pk, time.time() - start_time))
+        return rowcount
```

Comment:
I oppose this change for the following reasons:

- making changes here has crept out of the scope of this PR
- the `bulk_update` is not the only save occurring in this method, so returning only its row count isn't very idiomatic
- this does absolutely nothing to address the fact that `iobj.save(update_fields=computed_fields.keys())` is going to raise the `DatabaseError("Save with update_fields did not affect any rows.") exception in the case we care about